frontend/lib/norent/letter-builder/know-your-rights.tsx

```diff
@@ -4,49 +4,93 @@ import Page from "../../ui/page";
 import { Link } from "react-router-dom";
 import { AppContext } from "../../app-context";
 import { BackButton } from "../../ui/buttons";
+import { getUSStateChoiceLabels } from "../../../../common-data/us-state-choices";
 import {
-  getUSStateChoiceLabels,
-  USStateChoice,
-} from "../../../../common-data/us-state-choices";
-import { getNorentMetadataForUSState } from "./national-metadata";
+  StatePartnerForBuilderEntry,
+  getNorentMetadataForUSState,
+  assertIsUSState,
+  NorentMetadataForUSState,
+} from "./national-metadata";
 import { OutboundLink } from "../../analytics/google-analytics";
+import { getStatesWithLimitedProtectionsFAQSectionURL } from "../faqs";
+
+const StateWithoutProtectionsContent: React.FC<NorentMetadataForUSState> = (
+  props
+) => {
+  return (
+    <>
+      <p>
+        Unfortunately, we do not currently recommend sending a notice of
+        non-payment to your landlord. Sending a notice could put you at risk.{" "}
+        <Link to={getStatesWithLimitedProtectionsFAQSectionURL()}>
+          Learn more.
+        </Link>
+      </p>
+
+      <div className="content">
+        <p>
+          We’ve partnered with <PartnerLink {...props.partner} /> to provide
+          additional support.
+        </p>
+      </div>
```

Comment:
In the figma mock-ups, there was a third paragraph here that said ```If you’d still like to create an account, we can send you updates in the future.``` Was that removed?

Reply:
oops sorry forgot, just added it!